eng/common/TestResources/New-TestResources.ps1

```diff
@@ -351,6 +354,15 @@ try {
     # Enumerate test resources to deploy. Fail if none found.
     $repositoryRoot = "$PSScriptRoot/../../.." | Resolve-Path
     $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | Resolve-Path
+    if ($TestResourcesDirectory) {
```

Comment:
Do we need something similar in the clean-up script as well?

Reply:
I kept all resource naming convention and/or group name variables are still based off service directory to simplify cleanup.